lib/cli/file-set-pipeline/log.js

```diff
@@ -12,52 +12,43 @@
  * Dependencies.
  */
 
-var charm = require('charm')(process.stdout);
 var chalk = require('chalk');
-var logUpdate = require('log-update');
 var report = require('vfile-reporter');
 
 /**
+ * Check whether `file` is provided by the user.
+ *
+ * @param {File} file - Virtual file.
+ * @return {boolean?} - whether `file` is provided by the
+ *   user.
+ */
+function isProvidedByUser(file) {
```

Comment:
Okay ignoring it is fine, I added it hear because it makes the code look nicer (the filter can be on one line).

Also: great that you’re nitpicking ;)